### PR TITLE
[DOCS-6298] Condition builder and statement editor

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: process-automation/latest/using/index.md [hunk 1]
```diff
@@ -174,3 +174,87 @@ Once you have customized a filter, there are two options:
...
+
+* Click the **$** button to switch the condition builder to an expression editor where you can display and edit the JUEL expression.
+
+* Click the **+** button to add a new statement editor.
+
+* The available operators are:
+  * Every: All statements must be true for the condition to be evaluated as true.
+  * At least one: The condition will be evaluated to be true when one or more of the statements are true.
+  * None: If all the statements are false, then the condition will be evaluated to be true.
+  **Note:** The operator between statements is the same for all the statements.
```
Comment (pmartinezga) [inline process-automation/latest/using/index.md:192]: This note should be under the operators' bullets so you have to insert a new line and delete the spaces before
```suggestion

**Note:** The operator between statements is the same for all the statements.
```

**Review by pmartinezga — APPROVED** [review]
Just a minor format suggestion

**Review by pmartinezga — APPROVED** [review]

**Comment by oliveralfresco2018** [comment]
Test

**Review by pmartinezga — APPROVED** [review]

**Review by anxumalo — APPROVED** [review]
LGTM

You could add a glossary item for "JUEL" so it's clearer what's a JUEL expression.

**Pull request merged**